.github/workflows/main.yml

```diff
@@ -21,9 +21,8 @@ jobs:
 
       - name: Create a virtual environment
         run: git submodule init && git submodule update && python3 -m venv ./venv && . venv/bin/activate
-      - run: pip install --upgrade setuptools
-      - run: pip install -r requirements.txt
-      - run: pip install -e .
+      - run: python -m pip install build virtualenv
```

Comment:
Why does this action also install `virtualenv` but the `test-windows` does not?

Reply:
Ah good catch, ty. I'll remove that, we're not using virtualenv in the workflows.